partiql-eval/src/test/kotlin/org/partiql/eval/internal/PartiQLEngineDefaultTest.kt

```diff
@@ -264,6 +264,35 @@ class PartiQLEngineDefaultTest {
         assertEquals(expected, output)
     }
 
+    @OptIn(PartiQLValueExperimental::class)
+    @Test
+    fun testExclude() {
```

Comment:
Test succeeds when `@Disabled` is removed from L29.

https://github.com/partiql/partiql-lang-kotlin/blob/672a09d354f1cd0a5615d72b4192e2b07d7ad2f2/partiql-eval/src/test/kotlin/org/partiql/eval/internal/PartiQLEngineDefaultTest.kt#L29